### DISCUSSION TITLE
Presentation Rehearsals

### DISCUSSION THREAD

**Discussion by jfongli** [CSC207-2022F-UofT/course-project-studyhero#53] [General]
Hello! Our presentation date is coming up soon and I thought it'd be a nice thing to come up with dates of when to meet up in order to rehearse the presentation. The earlier we can schedule this, the better- that way we can see if we can book a room to practice in! 
You can fill out this when2meet: https://www.when2meet.com/?17927742-WlVlc

**Comment by madelahn** [comment]
Thanks for setting this up Jennifer! :D I think it's also a good idea to meet on Thursday before our presentation time as well, if we also wanted to discuss when everyone was available then.
